action.yaml

```diff
@@ -0,0 +1,33 @@
+name: 'Local Docker image scan'
+description: 'Scan Docker images using Trivy locally'
+inputs: 
+  image-name:
+    description: 'Docker image to scan'
+    required: true
+  whitelist-file:
+    description: 'Path to the whitelist file'
+    required: false
+  whitelist-file-branch:
```

Comment:
Should we remove this input?
In case of PR, we can find the target branch from the [event payload](https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context).
In other cases we should use whitelist from current branch.

We can currently assume this to be master and add the above logic later?

Reply:
I think we can add that logic later. For now, we can go ahead with an explicit input for the branch, which is going to be master in most cases.

Reply:
Sounds well.

Reply:
Shouldn't this always be the branch it is run against? 
You could get the value from `GITHUB_REF` variable.

Reply:
It's possible that different branches have different versions of the whitelist file, and the user might want to use a particular version. Although it makes more sense to use the whitelist file in the current branch. We can take PM input on this as well.